src/services/auth/AuthService.ts

```diff
@@ -418,4 +417,9 @@ export class AuthService {
 		// Update state in webviews once per unique controller
 		await Promise.all(Array.from(uniqueControllers).map((c) => c.postStateToWebview()))
 	}
+
+	private destroyTokens() {
+		this._controller.stateManager.setSecret("clineAccountId", undefined)
+		this._controller.stateManager.setSecret("cline:clineAccountId", undefined)
+	}
```

Comment:
Why not just use clearAuthToken if they are doing the same thing? Do we need to have both?